onert-micro/luci-interpreter/pal/mcu_best_mem/PALConv2d.h

```diff
@@ -0,0 +1,232 @@
+
+  // Create int16 input and output pointers for not USUAL status
+  const int16_t *int16_input_data = reinterpret_cast<const int16_t *>(input_data);
+  int16_t *int16_output_data = reinterpret_cast<int16_t *>(output_data);
+
+  for (int batch = 0; batch < batches; ++batch)
+  {
+    for (int out_y = 0; out_y < output_height; ++out_y)
+    {
+      const int in_y_origin = (out_y * stride_height) - pad_height;
+      for (int out_x = 0; out_x < output_width; ++out_x)
+      {
+        const int in_x_origin = (out_x * stride_width) - pad_width;
+        for (int out_channel = 0; out_channel < output_depth; ++out_channel)
+        {
+          float total = 0.f;
+          for (int filter_y = 0; filter_y < filter_height; ++filter_y)
+          {
+            const int in_y = in_y_origin + dilation_height_factor * filter_y;
+            for (int filter_x = 0; filter_x < filter_width; ++filter_x)
+            {
+              const int in_x = in_x_origin + dilation_width_factor * filter_x;
+
+              // Zero padding by omitting the areas outside the image.
+              const bool is_point_inside_image =
+                (in_x >= 0) && (in_x < input_width) && (in_y >= 0) && (in_y < input_height);
+
+              if (!is_point_inside_image)
+              {
+                continue;
+              }
+
+              for (int in_channel = 0; in_channel < input_depth; ++in_channel)
+              {
+                const int input_data_offset =
+                  ((batch * input_height + in_y) * input_width + in_x) * input_depth + in_channel;
+
+                const int filter_data_offset =
+                  ((out_channel * filter_height + filter_y) * filter_width + filter_x) *
+                    input_depth +
+                  in_channel;
+
+                float input_value = 0.f;
+                if (status == luci_interpreter::OperationGraphStatus::USUAL or
+                    status == luci_interpreter::OperationGraphStatus::START)
+                  input_value = input_data[input_data_offset];
+                else
+                  input_value =
+                    static_cast<float>(int16_input_data[input_data_offset]) * input_scale;
+
+                const float filter_value = filter_data[filter_data_offset];
+                total += (input_value * filter_value);
+              }
+            }
+          }
+          if (bias_data)
+          {
+            total += bias_data[out_channel];
+          }
+
+          const int output_data_offset =
+            ((batch * output_height + out_y) * output_width + out_x) * output_depth + out_channel;
+
+          total = std::min(std::max(total, output_activation_min), output_activation_max);
+
+          if (status == luci_interpreter::OperationGraphStatus::USUAL or
+              status == luci_interpreter::OperationGraphStatus::END)
+          {
+            output_data[output_data_offset] = total;
+          }
+          else
+          {
```

Comment:
supporting new kinds of operations status. This kernel has some of the differences from `pal/mcu` kernel